src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=no-self-use
```

Comment:
please do not use per-file disables. please disable only at the exact line of the violation